### ISSUE TITLE
saveAs not using file name with url

### ISSUE THREAD

**Issue by donhmorris** [eligrey/FileSaver.js#743]
Calling saveAs with a url as first parameter and file name as second parameter, the file name is not used when saving the file.

Example url:
https://firebasestorage.googleapis.com/v0/b/freightconn-8a4ee.appspot.com/o/users%2FLVkyPGhFVKPpUbHnoRDht7oWrSJ3%2Fbols%2FA_0h9YNdy-RqmyRdyroqM.afphoto?alt=media&token=0279f554-5472-4769-a7ab-f64ba1680eeb

**Comment by oreporan** [comment]
any fix? seems basic..

**Comment by Vishal-Isharani** [comment]
Same issue.

**Comment by wobushidaxingxing** [comment]
I have same problem with filename,different browers download the same file is different filename

**Comment by MaazKhawaja** [comment]
@donhmorris  have you found any workaround for this issue?

**Comment by donhmorris** [comment]
No. Completely forgot about this. I’ve had time to develop a fix since I reported this. I wish I hadn’t forgotten about it.

**Comment by rockmandash** [comment]
bro we need this

**Comment by voidsatisfaction** [comment]
Same here

**Comment by donhmorris** [comment]
I looked at the code. Too twisty for me. I don't really code in js. I'm a structured ts kind of guy.
